scripts/dts/extract/interrupts.py

```diff
@@ -51,6 +51,12 @@ def extract(self, node_address, yaml, prop, names, def_label):
         l_base = def_label.split('/')
         index = 0
 
+        # Newer versions of dtc might have the interrupt propertly look like
+        # interrupts = <1 2>, <3 4>;
+        # So we need to flatten the list in that case
+        if type(props[0]) is list:
```

Comment:
Minor codestyle suggestion: is more idiomatic in Python to use isinstance(foo, list). Current count of usages in scripts/dts/extract/*.py:

~~~
$ grep type *
reg.py:        if type(reg) is not list: reg = [ reg ]
$ grep isinstance *
clocks.py:        if not isinstance(properties, list):
default.py:        if isinstance(prop_values, list):
default.py:                if isinstance(prop_value, str):
default.py:            if isinstance(prop_values, str):
globals.py:    if isinstance(d, dict):
globals.py:        if isinstance(compat, list):
globals.py:    if isinstance(root, dict):
globals.py:    if isinstance(nodes, dict):
pinctrl.py:        if not isinstance(pinconf, list):
~~~

Reply:
Thanks for the review, will make the change.